테스트 수정 완료

Dae-Hwa opened the pull request:
- JUnit4 -> JUnit5로 수정
- ParameterizedTest 적용

sanhee committed:
fix: JUnit4로 적용되어 테스트 실행 안되는 문제 해결

기본 프로젝트의 import가 JUnit4로 되어 있음. JUnit5로 수정하고, assertJ 적용

Closes: #4

sanhee committed:
chore: ParameterizedTest 적용을 위해 junit-jupiter-params추가

ParameterizedTest가 현재 추가되어있는 디펜던시로는 사용할 수 없어서 새로운 디펜던시 추가. 상위 디펜던시에서 버전
명시가 되어있기 때문에 버전 생략

sanhee committed:
refactor: 작성 되어 있는 테스트 ParameterizedTest 이용하여 리팩토링

Closes: #5

sanhee committed:
chore: JUnit 의존성 간략화

기존 의존성은 [jupiter-api:5.6.0, jupiter-engine, jupiter-parms]로 3개였지만, 간략화하기
위해 이들 모두를 포함하는 junit-jupiter:5.6.0로 대체하였음.

sanhee commented on src/test/java/util/HttpRequestUtilsTest.java:
```diff
@@ -1,73 +1,133 @@
+                Arguments.of("userId=javajigi&password", "javajigi", null)
+        );
+    }
+
+
```
다른 곳은 줄바꿈이 한 칸인데, 여기만 줄바꿈이 두 칸으로 되어 있음
```suggestion

```

sanhee commented on src/test/java/util/HttpRequestUtilsTest.java:
```diff
@@ -1,73 +1,133 @@
+                Arguments.of("logined=true; JSessionId=1234", "JSessionId", "1234"),
+                Arguments.of("logined=true; JSessionId=1234", "session", null)
+        );
+    }
```
이 테스트는 다른 테스트와 통일성이 떨어지는 것 같음
argument가 중복되는데, 이렇게 하면 테스트 코드는 짧아지지만, 반대로 argument들이 불필요하게 중복될 수 있음.
그리고 테스트 코드에서 어떤 것을 조회할지 명시해주는게 더 직관적이라고 생각됨

sanhee replied:
수정 완료
900f795c5b0869e3f201a1c93fddaf7e517e6f4f

sanhee committed:
refactor: 통일성 없는 테스트케이스 수정

argument가 중복되는데, 이렇게 하면 테스트 코드는 짧아지지만, 반대로 argument들이 불필요하게 중복될 수 있음.
그리고 테스트 코드에서 어떤 것을 조회할지 명시해주는게 더 직관적이라고 생각됨.

sanhee commented on src/test/java/util/IOUtilsTest.java:
```diff
@@ -1,21 +1,33 @@
 
-public class IOUtilsTest {
+class IOUtilsTest {
     private static final Logger logger = LoggerFactory.getLogger(IOUtilsTest.class);
```
Logger가 사용되는 곳이 없어보임.

```suggestion
```

sanhee committed:
refactor: 불필요한 줄바꿈 제거

https://github.com/sanhee/java-was/pull/7#discussion_r600523331

sanhee committed:
refactor: 불필요한 멤버변수 제거

Logger가 쓰이는 곳이 없기 때문에 제거
https://github.com/sanhee/java-was/pull/7#discussion_r600544407

sanhee reviewed (approved):
👍
